robotbase/src/main/java/com/team2357/frc2023/subsystems/ArmExtensionSubsystem.java

```diff
@@ -0,0 +1,45 @@
+package com.team2357.frc2023.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import com.team2357.frc2023.Constants;
+import com.team2357.frc2023.Robot;
+
+import edu.wpi.first.wpilibj.RobotState;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class ArmExtensionSubsystem extends SubsystemBase{
+    private CANSparkMax m_armExtensionMotor;
+    public ArmExtensionSubsystem(){
+        m_armExtensionMotor = new CANSparkMax(Constants.CAN_ID.ARM_EXTENSION_MOTOR, MotorType.kBrushless);
+
+        m_armExtensionMotor.setSmartCurrentLimit(Constants.ARM_ROTATION.ARM_ROTATION_STALL_AMP_LIMIT, Constants.ARM_ROTATION.ARM_ROTATION_FREE_AMP_LIMIT);
```

Comment:
Why are you using `ARM_ROTATION` constants in `ArmExtensionSubsystem`? We will need to have separate constants for both subsystems.